packages/sampler/stories/default/Layout.js

```diff
@@ -29,7 +29,10 @@ export const _Layout = (args) => (
 			<Cell shrink={args['shrinkable cell']}>
 				<Item>Second</Item>
 			</Cell>
-			<Cell>
+			<Cell
+				grow={args['growable cell']}
```

Comment:
It's quite ambiguous that `grow` control matches this Cell. Let's rename it clearer. Please do the same to `shrink`

Reply:
I'll fix it.